wordle-web/pages/game.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-row justify="center" align="center">
+    <v-col cols="12" sm="8" md="6">
+        <v-card-title class="headline justify-center">
+          <h1>Wordln't</h1>
+        </v-card-title>
+      <v-card class="logo py-4 d-flex justify-center">
+        <DTCannon />
+      </v-card>
+      <v-card>
+        <v-card-title class="headline">
+          Welcome to the game page.
+        </v-card-title>
+        <v-card-text>
+          <p>
+            This game page uses a progressive Material Design component framework for
+            Vue.js. 
+          </p>
```

Comment:
No need to wrap this in a `<p>` tag! The `<v-card-text>` should take care of that for you.